components/Pagination/sub-components/Desktop.jsx

```diff
@@ -35,6 +36,7 @@ const Desktop = ({
 
       return (
         <PageButton
+          tabIndex={0}
```

Comment:
This value always be zero? Is your order already defined relative to the other elements?

Reply:
@MichelAraujo now the childrens `tabIndex` prop is inherited from the parent component.
See f9fdfe75b894115005a9f410d3b72265fa169f04.